src/pyodide/internal/workers.py

```diff
@@ -197,10 +198,19 @@ def _to_js_headers(headers: Headers):
 
 
 class Response(FetchResponse):
+    """
+    This class represents the response to an HTTP request, with a similar API to that of the web
+    `Response` API: https://developer.mozilla.org/en-US/docs/Web/API/Response.
+
+    This is a static class to enable defining `FetchResponse` using a constructor more similar to
+    JS. All non-static methods for this class should be defined on the `FetchResponse` class it
+    extends.
```

Comment:
Why are `Response` and `FetchResponse` separate classes?

Reply:
Hmm, I don't recall why. I guess we can try and merge them in a separate PR.